lib/interactive.dart

```diff
@@ -42,6 +55,22 @@ class Interactive {
     _loadFilters();
   }
 
+  num _gatherChartWidth() {
```

Comment:
Consider `_computeChartWidth` - gather seems an unusual verb to use here, though give what you need to do I certainly understand how you arrived at it

Reply:
Fixed